README.md

```diff
@@ -212,8 +206,9 @@ You can find example of drivers [here](https://github.com/RonasIT/laravel-swagge
 ### Viewing OpenAPI documentation
 
 As of version 2.2, the package includes the ability to switch between OpenAPI documentation
-viewers. To access different viewers, modify the `documentation_viewer` configuration.
-This change is reflected immediately, without the need to rebuild the documentation file.
+viewers. To access different viewers, modify the `documentation_viewer` configuration or set the viewer using
+the `SWAGGER_SPEC_VIEWER` env. This change is reflected immediately, without the need to rebuild the documentation file.
+The available viewers are: "swagger", "elements", "rapidoc"
```

Comment:
no reasons to duplicate the available values here, it will require to update the available viewers list in two places

```suggestion
```